install-dev/langs/de/data/tab.xml

```diff
@@ -1,51 +1,51 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <entity_tab>
-  <tab id="SELL" name="SELL" />
-  <tab id="IMPROVE" name="IMPROVE" />
-  <tab id="CONFIGURE" name="CONFIGURE" />
-  <tab id="DEFAULT" name="DEFAULT" />
+  <tab id="SELL" name="VERKAUFEN" />
+  <tab id="IMPROVE" name="VERBESSERN" />
+  <tab id="CONFIGURE" name="VERWALTEN" />
+  <tab id="DEFAULT" name="STANDARD" />
   <tab id="Addresses" name="Adressen"/>
-  <tab id="Module_Page" name="Modules &amp; Services" />
+  <tab id="Module_Page" name="Module &amp; Dienstleister" />
   <tab id="Administration" name="Verwaltung"/>
   <tab id="Advanced_Parameters" name="Erweiterte Einstellungen"/>
   <tab id="Attachments" name="Anhänge"/>
   <tab id="Attributes_and_Values" name="Attribute &amp; Gruppen"/>
-  <tab id="Attributes_and_features" name="Attributes &amp; Features" />
+  <tab id="Attributes_and_features" name="Attribute &amp; Eigenschaften" />
   <tab id="CMS_Categories" name="CMS-Kategorien"/>
   <tab id="CMS_Pages" name="CMS-Seiten"/>
   <tab id="CSV_Import" name="Import"/>
-  <tab id="CarrierWizard" name="Carrier"/>
+  <tab id="CarrierWizard" name="Versanddienst"/>
   <tab id="Carriers" name="Versanddienste"/>
   <tab id="Cart_Rules" name="Warenkorb Preisregeln"/>
   <tab id="Catalog" name="Katalog"/>
   <tab id="Catalog_Price_Rules" name="Katalog Preisregeln"/>
   <tab id="Categories" name="Kategorien"/>
   <tab id="Combinations_Generator" name="Kombinationsgenerator"/>
   <tab id="Configuration" name="Konfiguration"/>
-  <tab id="Contact" name="Contact" />
+  <tab id="Contact" name="Kontakt" />
   <tab id="Contacts" name="Kontakte"/>
   <tab id="Countries" name="Länder"/>
   <tab id="Credit_Slips" name="Gutschriften"/>
   <tab id="Currencies" name="Währungen"/>
   <tab id="Customer_Service" name="Kundenservice"/>
-  <tab id="Customer_Service_2" name="Customer Service" />
-  <tab id="Customer_settings" name="Customer Settings" />
+  <tab id="Customer_Service_2" name="Kundenservice" />
+  <tab id="Customer_settings" name="Kunden" />
```

Comment:
Shouldn't we say "Kundeneinstellungen" or "Kunden Einstellungen" ?
Otherwise you'd have 2 pages with the same title (see lines 33 to 35).